apps/cli-daemon/src/app/workspace-manager/workspace-manager.controller.ts

```diff
@@ -24,4 +24,9 @@ export class WorkspaceManagerController {
       throw new InternalServerErrorException();
     }
   }
+
+  @Get('home')
```

Comment:
I'm just not sure about the endpoint name, maybe home-directory? "home" feels to general

Reply:
maybe 'homedir'